docs/cmdline-opts/fail.d

```diff
@@ -1,15 +1,15 @@
 Long: fail
 Short: f
 Protocols: HTTP
-Help: Fail silently (no output at all) on HTTP errors
+Help: Fail fast with no output on HTTP errors
```

Comment:
Is "Fail fast" actually true, it doesn't really fail faster than any other way (apart from not outputting) or than the server allows it to. I don't have a better idea off the cuff, but fast seem so imply something it doesn't do.

Reply:
It does fail "fast" in the way that it stops reading the response after the headers if the response code tells it to.

Reply:
Fair.